src/callbacks_step/analysis.jl

```diff
@@ -141,6 +141,49 @@ function AnalysisCallback(mesh, equations::AbstractEquations, solver, cache;
                      initialize = initialize!)
 end
 
+function AnalysisCallback(mesh, equations::AbstractEquations, solver::FV, cache;
+                          interval = 0,
+                          save_analysis = false,
+                          output_directory = "out",
+                          analysis_filename = "analysis.dat",
+                          extra_analysis_errors = Symbol[],
+                          analysis_errors = union(default_analysis_errors(equations),
+                                                  extra_analysis_errors),
+                          extra_analysis_integrals = (),
+                          analysis_integrals = union(default_analysis_integrals(equations),
+                                                     extra_analysis_integrals),
+                          RealT = real(solver),
+                          uEltype = eltype(cache.elements[1]), # TODO: this routine is equal to the existing one except this default type here.
```

Comment:
The existing routine with `solver::dg` uses `uEltype = eltype(cache.elements)` as the default uEltype. Unfortunately, this is not possible here since elements is a vector consisting of containers for every element by itself.
That's why I created an additional routine.
For the future, the `elements` structure in cache is probably not mergeable and should be changed. I added this comment here to keep the issue in mind.

Reply:
Solved with restructuring in [72573f8](https://github.com/trixi-framework/Trixi.jl/pull/1844/commits/72573f83dc2133156a3c72e8343736027cd20770)